reference/forms/types/text.rst

```diff
@@ -44,7 +44,9 @@ These options inherit from the :doc:`FormType </reference/forms/types/form>`:
 .. include:: /reference/forms/types/options/empty_data.rst.inc
     :end-before: DEFAULT_PLACEHOLDER
 
-The default value is ``''`` (the empty string).
+From an HTTP perspective, submitted data is always a string or an array of strings.
+So by default, the form will treat any empty string as null. If you prefer to get
+an empty string, explicitly set the ``empty_data`` option to an empty string.
```

Comment:
@symfony/team-symfony-docs should we care about adding a note to say this possible since 3.2 only?